stdlib/typing_extensions.pyi

```diff
@@ -569,8 +562,70 @@ else:
     ReadOnly: _SpecialForm
     TypeIs: _SpecialForm
 
+# TypeAliasType was added in Python 3.12, but had significant changes in 3.14.
+if sys.version_info >= (3, 14):
+    from typing import TypeAliasType as TypeAliasType
+else:
+    @final
+    class TypeAliasType:
+        def __init__(
+            self, name: str, value: Any, *, type_params: tuple[TypeVar | ParamSpec | TypeVarTuple, ...] = ()
+        ) -> None: ...  # value is a type expression
+        @property
+        def __value__(self) -> Any: ...  # a type expression
+        @property
+        def __type_params__(self) -> tuple[TypeVar | ParamSpec | TypeVarTuple, ...]: ...
+        @property
+        def __parameters__(self) -> tuple[TypeVar | ParamSpec, ...]: ...
```

Comment:
is it not possible for a `__parameters__` element to be a `TypeVarTuple`?

```suggestion
        def __parameters__(self) -> tuple[TypeVar | ParamSpec | TypeVarTuple, ...]: ...
```

Reply:
A `TypeVarTuple` gets expanded:

https://github.com/python/typing_extensions/blob/478b2b366beb30d74d5dd0029848141bf911db7f/src/typing_extensions.py#L3924-L3927

Reply:
Ah, indeed. Which shows us that the `__parameters__` tuple can include special forms as well as `TypeVar`s and `ParamSpec`s:

```pycon
>>> from typing_extensions import *
>>> Ts = TypeVarTuple("Ts")
>>> A = TypeAliasType("A", int, type_params=(Ts,))
>>> A.__parameters__
(typing.Unpack[Ts],)
>>> A.__parameters__[0].__class__
<class 'typing._UnpackGenericAlias'>
```

It works the same using the `type` statement:

```pycon
>>> type X[*Ts] = T
>>> X.__parameters__
(typing.Unpack[Ts],)
```

So this should perhaps (unfortunately) just be:

```suggestion
        def __parameters__(self) -> tuple[Any, ...]: ...
```

Reply:
or perhaps, for a little more type safety:

```suggestion
        def __parameters__(self) -> tuple[TypeVar | ParamSpec | Any, ...]: ...
```